framework/di/Container.php

```diff
@@ -456,48 +457,68 @@ protected function resolveDependencies($dependencies, $reflection = null)
     }
 
     /**
-     * Invoke callback with resolved dependecies parameters.
+     * Resolve dependencies for a function, can use either named or numbered $params.
```

Comment:
Can we come up with a better description here? For me it's not clear what the method does from the description.